index.js

```diff
@@ -57,10 +62,7 @@ program
     })
     .parse(process.argv);
 
-
-// windows only
-var isWinAdmin;
-
+var isWinAdmin; // windows only
```

Comment:
I think this variable's name and use can be changed.
1) `isWinAdmin` seems to suggest a boolean, but is actually a function
2) I'm not sure at what other points this would be used, but if the only point of use is on line:78, then you could just do the dynamic import there and be done with it. Otherwise I don't think it's terrible to just directly do a require instead of trying to dynamically assign it to `require('is-administrator')` -- does that make sense?